messaging/kafka-avro-reactive-messaging/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>kafka-avro-reactive-messaging</artifactId>
+
+    <name>Quarkus OpenShift TS: Messaging: kafka-avro-reactive-messaging</name>
```

Comment:
Better use a more descriptive name like in the rest of the modules.
For example, Quarkus OpenShift TS: Messaging: Kafka with Avro